CMakeLists.txt

```diff
@@ -497,7 +497,7 @@ if(NOT USE_SYSTEM_NATPMP)
             NATPMP_STATICLIB)
 endif()
 
-tr_add_external_auto_library(MINIUPNPC miniupnpc miniupnpc
+tr_add_external_auto_library(MINIUPNPC miniupnp/miniupnpc miniupnpc
     TARGET miniupnpc::libminiupnpc
```

Comment:
Could be `lib` suffix in `miniupnpc::libminiupnpc` the issue failing Windows builds?

@Coeur I suggest you add `tree <cmake_build_dir>\third-party\miniupnp\miniupnpc.bld\pfx` to windows build action to inspect layout of what is being built for `miniupnp` target.

Reply:
I have a Windows machine to test locally, will update if I found anything.

Reply:
@nevack you'd need to tell me in which file and which line, because I'm confused by what is that `tree` command. Is it for CMakeLists.txt or for actions.yml or something else?

Reply:
Also, related but unrelated, Windows builds are already failing at the moment, for another reason: https://github.com/transmission/transmission/issues/6680

Reply:
Quick and dirty fix, but it at least builds successfully.

<details>
<summary>patch</summary>

```patch
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 27c305291..5e0bf64d1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -497,7 +497,14 @@ if(NOT USE_SYSTEM_NATPMP)
             NATPMP_STATICLIB)
 endif()
 
-tr_add_external_auto_library(MINIUPNPC miniupnp/miniupnpc miniupnpc
+if(WIN32)
+    # https://github.com/miniupnp/miniupnp/pull/304
+    # Microsoft please (╬▔皿▔)╯
+    set(TR_MINIUPNPC_LIBNAME libminiupnpc)
+else()
+    set(TR_MINIUPNPC_LIBNAME miniupnpc)
+endif()
+tr_add_external_auto_library(MINIUPNPC miniupnp/miniupnpc ${TR_MINIUPNPC_LIBNAME}
     TARGET miniupnpc::libminiupnpc
     CMAKE_ARGS
         -DUPNPC_BUILD_STATIC=ON
@@ -511,6 +518,7 @@ if(NOT USE_SYSTEM_MINIUPNPC)
     set(MINIUPNPC_VERSION 1.9)
     set(MINIUPNPC_API_VERSION 12)
 endif()
+unset(TR_MINIUPNPC_LIBNAME)
 
 target_compile_definitions(miniupnpc::libminiupnpc
     INTERFACE
```

</details>

Reply:
I find it really funny that it's indeed `lib` prefix after all 😄

Reply:
@tearfur btw, why is CMake not picking up upstream fix in the PR you mentioned inside patch comment?

Reply:
[296b69e](https://github.com/transmission/transmission/pull/6459/commits/296b69eedd8eafc06432402bd278fc6921319297)